### PR TITLE
Update index.html.md.erb

### PR DESCRIPTION
**Pull request by Ciandelle**
Add detailed descriptions of designers abilities in this section of the team documentation

### CODE REVIEW

File: source/design-and-content/index.html.md.erb [hunk 1]
```diff
@@ -3,4 +3,24 @@ title: Design and content
 weight: 12
 ---
 
-# Design and content
\ No newline at end of file
+# Design and content
+
+## Designers on the team
+
+### Chris
```
Comment (christopherthomasdesign) [inline source/design-and-content/index.html.md.erb:10]: I think this should probs just be in alphabetical order tbh!

**Review by christopherthomasdesign — CHANGES_REQUESTED** [review]
I don't think the content's been removed from the page it was on before (guides/get-help-from-other-roles)

File: source/design-and-content/index.html.md.erb (hunk 1 above)
Comment (Ciandelle) [reply source/design-and-content/index.html.md.erb:10]: Cool, I can get it changed, and for some reason I had to do 2 separate requests :(

**Review by Ciandelle — COMMENTED** [review]
Happy with the changes

**Review by CharlotteDowns — APPROVED** [review]

File: source/design-and-content/index.html.md.erb (hunk 1 above)
Comment (stevenjmesser) [reply source/design-and-content/index.html.md.erb]: Resolving this conversation as Ciandelle addressed it in another commit.

**Comment by stevenjmesser** [comment]
I think @christopherthomasdesign has to give it another review.

**Comment by stevenjmesser** [comment]
(Piggybacking on this PR to add a small update to the README.)

**Review by christopherthomasdesign — APPROVED** [review]
Thank for doing this! Looks good

**Pull request merged**